### DISCUSSION TITLE
OPNpool with M5stack?

### DISCUSSION THREAD

**Discussion by ctillier-vmware** [cvonk/OPNpool#6] [Q&A] [answered]
Hi Coert,
I'm new to the ESP32 and have been playing with the M5 stack to learn. I got a couple of the different boards that integrate plug and play to 485... So I was wondering if you have tried them? 
I  got the M5Stamp (https://docs.m5stack.com/en/stamp/stamp_pwr485) that has a minimalist approach to 485 communication but still has the main component 485 control that powers the board...

I'll try a couple of things, and let you know if I can get it to integrate with the Pentair system I have.
Thanks,
Chris

**Comment by cvonk** [comment, marked as answer]
There are many boards and kits that incorporate the ESP32.  I chose mine six years ago based on the ESP8266 and later bumped it up to the ESP32.  My system has been running ever since. 

If you want to try something else, sure feel free.  Just make sure you use the MAX3485 instead of the TTL-level MAX485.

/c
